Types.ts

```diff
@@ -20,7 +20,7 @@ export type LinkPublicKeyConfiguration = (CommonPlaidLinkOptions & {
     userEmailAddress?: string;
     userPhoneNumber?: string;
     linkCustomizationName?: string;
-    accountSubtypes?: Array<LinkAccountSubtype>;
+    accountSubtypes?: Array<LinkAccountSubtypeInterface>;
```

Comment:
Should we rename this to `LinkAccountSubtype`?

```suggestion
    accountSubtypes?: Array<LinkAccountSubtype>;
```

This is part of our public interface and for our customers this is what it actually represents.

Since `LinkAccountType` and `LinkAccountSubType` are private to our javascript module, we can name them either `AccountType` or `LinkAccountTypeEnum`.